src/Integrations/OpenAi/ProductMapper.php

```diff
@@ -218,7 +218,7 @@ private function get_enable_with_override( \WC_Product $product, string $meta_ke
 	 */
 	protected function get_enable_search( \WC_Product $product, ?\WC_Product $parent_product ): string {
 		// For variations, check parent product meta; for simple products, check product meta.
-		$check_product = $parent_product ? $parent_product : $product;
+		$check_product = $parent_product ?? $product;
```

Comment:
Good point. Before stronger types, we were at risk of `false ?? $product` resulting in `false`, but we have strong types now!